### PR TITLE
Added books and courses

### PR DESCRIPTION
**Pull request by lemonsaurus**
Since the Andrew PR was closed due to inactivity, I took it upon myself to finish this myself. New resources page looks like this:

image

image

image

image

### CODE REVIEW

File: static/resources.json [hunk 1]
```diff
@@ -1,22 +1,117 @@
...
+          {
+            "title": "Crafting Quality Code",
+            "url": "https://www.coursera.org/learn/program-code",
+            "icon": "regular/graduation-cap"
+          }
+        ],
+        "payment": "optional",
+        "payment_description": "You can pay to enroll for a graded certificate, or choose to audit for free."
+      },
+      "MIT: Introduction to Computer Science and Programming Using Python": {
```
Comment (gdude2002) [inline static/resources.json]: Please alphabetize.

File: static/resources.json [hunk 2]
```diff
@@ -35,77 +130,166 @@
...
+            "icon": "branding/github"
           }
         ],
-        "payment": "free",
-        "payment_description": null
+        "payment": "paid"
       },
-      "A Simple Guide to Git": {
-        "description": "A simple, no-nonsense guide to the basics of using Git.",
+      "Effective Python": {
```
Comment (gdude2002) [inline static/resources.json]: Please alphabetize.

File: static/resources.json [hunk 3]
```diff
@@ -35,77 +130,166 @@
...
+            "icon": "branding/github"
           }
         ],
-        "payment": "free",
-        "payment_description": null
+        "payment": "paid"
       },
-      "Python Tutorials by Corey Schafer on YouTube": {
-        "description": "An in-depth look at the Python programming language, from one of YouTube's most popular Python tutors.",
+      "Flask Web Development": {
```
Comment (gdude2002) [inline static/resources.json]: Please alphabetize.

File: static/resources.json [hunk 4]
```diff
@@ -186,7 +358,19 @@
         ],
         "payment": "free",
         "payment_description": null
+      },
+      "PyCharm": {
```
Comment (gdude2002) [inline static/resources.json]: Please alphabetize.

**Review by gdude2002 — CHANGES_REQUESTED** [review]
Please alphabetize.

**Comment by gdude2002** [comment]
Please alphabetize.

**Comment by lemonsaurus** [comment]
Alphabetize within payment type or overall? because I guess the former is okay but the latter will look terrible. It should be free, optional, paid.

**Comment by gdude2002** [comment]
It's always been entirely alphabetized, so I haven't been thinking about how the icons look that way. I think I agree.

**Review by gdude2002 — DISMISSED** [review]

**Comment by jchristgit** [comment]
I like the overall design, but I feel like the buttons below the names look kinda out of place. 
For example, consider this:
image
Other university courses on the page only have a single button linking to what I assume to be the course page. While the description here says "a 2-part-course", it may become clear that the first button is the first part and the second button the second part, but what about e.g. the "5-part specialization"? Maybe adding some description next to the buttons would be great, for example putting the `title` of the button there. Would be happy for other opinions on this.

**Comment by lemonsaurus** [comment]
@Volcyy if you hover the buttons, there is a description explaining exactly this.

**Comment by gdude2002** [comment]
I have nothing against putting actual text in the buttons. The buttons were my idea, btw.

**Comment by lemonsaurus** [comment]
New order:

image

image

**Review by gdude2002 — APPROVED** [review]

**Pull request merged**